chart/templates/cronjob.yaml

```diff
@@ -27,14 +30,20 @@ spec:
           restartPolicy: Never
           securityContext:
             runAsNonRoot: true
+            {{ if .Values.serviceUser.uid }}
+            runAsUser: {{ .Values.serviceUser.uid }}
```

Comment:
The documentation says this should be `.Values.user` rather than serviceUser, let's go with the former?

Reply:
The values.yaml also has serviceUser.